tests/NCronJob.Tests/RuntimeJobRegistryTests.cs

```diff
@@ -92,7 +134,7 @@ public async Task ShouldThrowAnExceptionWhenJobIsNotFoundAndTryingToUpdateSchedu
         await provider.GetRequiredService<IHostedService>().StartAsync(CancellationToken);
         var registry = provider.GetRequiredService<IRuntimeJobRegistry>();
 
-        Should.Throw<InvalidOperationException>(() => registry.UpdateSchedule("JobName", "* * * * *"));
+        Should.Throw<InvalidOperationException>(() => registry.UpdateSchedule("JobName", AtEveryMinute));
```

Comment:
@linkdotnet If those changes are too noisy for you in the context of that PR, just let me know and I'll happily drop them.

Beware that I might try to reintroduce them in a future PR though, as I find those variables slightly easier to parse than crude cron expressions :wink:

Reply:
No. I like it!